pkg/network/encoding/format.go

```diff
@@ -233,9 +236,20 @@ func routeKey(v *network.Via) string {
 	return v.Subnet.Alias
 }
 
-func formatTags(tagsSet *network.TagsSet, c network.ConnectionStats) (tagsIdx []uint32) {
+func formatTags(tagsSet *network.TagsSet, c network.ConnectionStats) (tagsIdx []uint32, checksum uint32) {
+	csum := murmur3.New32()
```

Comment:
This will work if we can guarantee that the same set of tags will always be in the same order each time.  If the order can change, we want to XOR the individual hashes together

Reply:
The static tag ordering is guaranteed, but dynamic tags will not be. I updated to use XORing.